Dockerfile

```diff
@@ -37,7 +37,7 @@ WORKDIR /app
 # This is needed to download transitive dependencies instead of compiling them
 # https://github.com/montanaflynn/golang-docker-cache
 # https://github.com/golang/go/issues/27719
-# renovate: datasource=repology depName=alpine_3_21/bash versioning=loose
+# renovate: datasource=repology depName=alpine_3_22/bash versioning=loose
```

Comment:
[`golang:1.25.4` uses Alpine 3.22](https://hub.docker.com/layers/library/golang/1.25.4-alpine/images/sha256-154a3d4ba5a80316771e042c706e5e0f84e01e80d3ccba6bc7df6ea93702e93c) so I thought I should update this comment too. There was no need to update the `bash` version as the same [`5.2.37-r0` is used in Alpine 3.22](https://pkgs.alpinelinux.org/packages?name=bash&branch=v3.22&repo=&arch=x86_64&origin=&flagged=&maintainer=).